src/main/java/graphql/execution/preparsed/persisted/ApolloPersistedQuerySupport.java

```diff
@@ -47,4 +53,18 @@ public ApolloPersistedQuerySupport(PersistedQueryCache persistedQueryCache) {
         }
         return Optional.empty();
     }
+
+    @Override
+    protected boolean persistedQueryIdIsInvalid(Object persistedQueryId, String queryText) {
```

Comment:
I would love to some javadoc here to tell other implementors of the contratc expected